kumascript/tests/macros/fixtures/apiref/subpages.json

```diff
@@ -76,7 +76,7 @@
   },
 
   {
-    "tags": ["Deprecated", "Non-standard", "Obsolete", "Experimental"],
+    "tags": ["Deprecated", "Non-standard", "Experimental"],
```

Comment:
This change is realistic - Obsolete should not be used. But I am mostly doing to get help/advice on how to update the test code here?

Further, who do I speak to for linting to make use of the status rather than tags to be caught?

Reply:
For linting, the general topic is "linting/validating frontmatter", and it would be good to open an issue for it (if it doesn't exist yet).

Reply:
Done in https://github.com/mdn/yari/issues/7900

I've taken a first shot at what is required - @wbamberg you might want to look at that too - feel free to directly edit my "requirements".